src/guide/develop/quick-start/generic-message.md

```diff
@@ -18,40 +18,54 @@ To create the job, you'll need to define the following parameters:
 
 Here is an example for creating a job called demo to be executed on Ethereum Mainnet for [this example contract](https://etherscan.io/address/0x51eca2efb15afacc612278c71f5edb35986f172f). In this case we want to update the storage variable to 234.
 
-**CLI command**
+#### Define the def.json
 ```sh
-palomad tx create job --job-id demo -chain-type evm --chain-ref-id eth-main --definition /tmp/def.json --payload /tmp/payload.json --payload-modifiable=true --from <your paloma address> --gas auto --fees 300ugrain --b block
-```
-
-**Definition JSON**
-```json
+cat << EOF > def.json
```

Comment:
We don't actually need this `cat << ...` syntax unless we're defining the variable inline. Here it would be fine to leave it out (especially since this code block is labeled `\`\`\`json`.